Add mapbox zoom 1793 v2

DrAcula27 opened the pull request:
Fixes #1793 

  - [x] Up to date with `main` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [x] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

ryanfchase commented on components/Map/zoomTooltip.jsx:
```diff
@@ -0,0 +1,39 @@
+              neighborhood council.
+            </strong>
+            <br />
+            To reset zoom features, please exit by clicking out of the
```
We needed to change this to say "please exit by clicking the 'x' on the selected NC" or something like that. More discussion incoming at upcoming general meeting

DrAcula27 replied:
I updated the text to: `To reset zoom features, please exit the boundary selection by clicking the 'X' on the selected Neighborhood Council within the 'Boundaries' filter of the 'Search & Filters' modal.`

ryanfchase reviewed (changes requested):
Mapbox `fitBounds` is behaving poorly. Steps to reproduce:

1. select an NC (preferably small, e.g. "Greater Valley Glen")
2. select another NC (preferably bigger, e.g. "Van Nuys NC")
3. notice that we don't get the smooth transition between selected NCs

Steps to fix: 

- in `components/Map/layers/BoundaryLayer.js`, notice that there is a call to `this.map.fitBounds()` on line ~205
  - *just above that line*, set the map's min zoom to our default min zoom
- optionally: move DEFAULT_MIN_ZOOM and DEFAULT_MAX_ZOOM into the constants folder (so we don't need to hard-code it in 2 places)

Example code starting at line ~203 in BoundaryLayer
```js
    // zoom to the region
    this.map.setMinZoom(9) // here! 9 is DEFAULT_MIN_ZOOM
    this.map.fitBounds(boundingBox(geo), { padding: FIT_BOUNDS_PADDING }); // this was already here
```

DrAcula27 commented:
@ryanfchase @traycn 
I have fixed the bug @ryanfchase mentioned in the previous comment.
- I moved the `DEFAULT_MIN_ZOOM` and `DEFAULT_MAX_ZOOM` constant variables from `Map.jsx` to the `CONSTANTS.js` file.
- at line 205 in `BoundaryLayer.js` I added `this.map.setMinZoom(DEFAULT_MIN_ZOOM)`
